paasta_tools/autoscale_cluster.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# Copyright 2015 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import argparse
+import logging
+
+from paasta_tools.autoscaling_lib import autoscale_local_cluster
+
+
+log = logging.getLogger('__main__')
```

Comment:
Hmm. We don't do this Null trick with other included libraries (marathon_tools), should we?

But when I do add the null handler, when I try to log from inside the library, I get this:

    AttributeError: 'NoneType' object has no attribute 'debug'